Fix error message location to be consistent

Nothell opened the pull request:
<!-- Describe your pull request here.-->
Closes [issue 17571](https://github.com/zulip/zulip/issues/17571).

 I have made the error message location on the organization settings section consistent by showing error messages beside the heading.
 For eg: Showed the error message **File type is not supported** beside the heading **Organization profile picture**. 


**Screenshots and screen capture:**

File Type Error:
![filetype-error](https://user-images.githubusercontent.com/33905827/222944694-ee7320f5-5f16-461c-82cb-95af7940eb1d.png)

Size more than 5MB error:
![Morethan5](https://user-images.githubusercontent.com/33905827/222944737-0cfa7370-c1aa-450c-9b69-055595ceb41d.png)

File Type Error beside Organization Logo Heading
![filetype-org-logo-dark](https://user-images.githubusercontent.com/33905827/222944787-784d8886-11cd-4454-a129-3124ec357860.png)

personal profile picture error location:
![image](https://user-images.githubusercontent.com/33905827/222944835-31ae98ee-2938-48ab-ba5c-8306e0b0772e.png)


<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Testing plan:
Manually tested in the browser

alya commented:
@Nothell Your PR will be reviewed once it passes the tests. Thanks!

alya commented:
You will also need to clean up your commit history. See [here](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for guidelines.

Nothell commented:
> You will also need to clean up your commit history. See [here](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for guidelines.

Nothell commented:
Hi @alya, Thank you for the guide. I have made changes as requested.

alya commented:
Thanks! The styling of the personal profile picture error does not look the same as the others.

alya commented:
The location of the org profile picture error is not as described in the issue, but might be fine -- putting it next to the section heading may end up making it scrolled off-screen.

alya commented:
The error styling also does not look right in dark theme:

<img width="535" alt="Screen Shot 2023-03-06 at 11 55 07 PM" src="https://user-images.githubusercontent.com/2090066/223359472-5a887703-6eb6-4c35-991c-a00297889d09.png">

alya commented:
Please make the above changes, re-add the checklist in the PR template, and carefully go through all the manual testing points.

Nothell commented:
Thank you for the feedback. I would like to take a moment to explain the reasoning behind the changes I made.

 1) > Thanks! The styling of the personal profile picture error does not look the same as the others.
Based on the note added here https://github.com/zulip/zulip/issues/17571#issue-829318527, I chose not to alter the existing styling of the personal profile picture error in order to ensure consistency throughout the personal settings section.

2) > The location of the org profile picture error is not as described in the issue, but it might be fine -- putting it next to the section heading may make it scrolled off-screen.
I have followed the specific notes added to the issues comment section here which specifies the location of the org profile picture error https://github.com/zulip/zulip/issues/17571#issue-829318527 and confirmed my changes with the PR comment here https://github.com/zulip/zulip/pull/20385#issuecomment-987042886. 


3) > The error styling also does not look right in the dark theme:
Will fix this issue

For the first two points if you have any further suggestions or concerns, please let me know.

alya commented:
> 1. Based on the note added here [Invalid Image upload UI element Bug #17571 (comment)](https://github.com/zulip/zulip/issues/17571#issue-829318527), I chose not to alter the
>    existing styling of the personal profile picture error in order to ensure consistency throughout the personal settings section.

Sorry, what comment? The link just goes to the issue for me.

Can you add screenshots of an error message or two that this currently matches?

alya commented:
Your other links to specific comments are not working for me either. If you are making decisions that are not obvious by looking at the issue, you should add detailed the explanations to the PR description; the reviewer shouldn't have to spelunk comments, PRs, etc., to understand the changes.

Nothell commented:
Ok noted! Here is the screenshot of the comments mentioned in the issue:

![image](https://user-images.githubusercontent.com/33905827/223499142-f4c195d9-0b23-4cb4-985e-d60cd54302d7.png)

alya commented:
OK, I don't see anything about that styling in that note.

Nothell commented:
I will do my best to provide further clarification.

>Thanks! The styling of the personal profile picture error does not look the same as the others.

Based on the note added in issue #17571, which I have included a screenshot of for your reference, I understand that there is some ambiguity regarding the styling of the personal profile picture error message. While the comment suggests that it is "probably appear below the error," it also recommends making it consistent with other messages, which is somewhat unclear.
![3rd point](https://user-images.githubusercontent.com/33905827/223883568-9af1894b-72c7-42dd-8b5c-f1a4da75956c.png)

To address this, I am considering adding the error message to the right of the Personal Profile picture, which is currently located below it. However, doing so would require me to move the Profile picture image to the left to ensure that the error message does not go outside the viewport.
 
I have included a screenshot to help illustrate what I am trying to explain. Thank you for your understanding, and please let me know if you have any further questions or concerns.
![2ndpoint_diagram](https://user-images.githubusercontent.com/33905827/223896178-57931b15-5ea3-4343-bd7a-61cee731e719.png)


> The location of the org profile picture error is not as described in the issue, but it might be fine -- putting it next to the section heading may make it scrolled off-screen

Regarding the location of the org profile picture error, I noticed your note in the issue stating that placing the error next to the "Organization profile picture" section heading would be a better idea, so that's why I made that change.
![2ndpoint](https://user-images.githubusercontent.com/33905827/223885508-85c4de99-d904-4e88-9ca9-12012b8aa2db.png)

>The error styling also does not look right in the dark theme:
  Will raise PR for this in a while

alya commented:
I don't think we should move the profile picture just for the error message. But while keeping the placement below, why can't we make the font/colors the same as other error messages?

Nothell commented:
I have used the same font and colours as other error messages for personal settings. Please take a look at the screenshot below. 
![3rdpointcss](https://user-images.githubusercontent.com/33905827/225455114-8bd4d8bd-c881-4098-9001-5900df9db17c.png)

alya commented:
Please add the following:

- A dark theme screenshot of the error in the comment above
- Screenshots of other errors in personal settings in (dark and light theme)

Thanks!

Nothell commented:
I added screenshots as requested and fixed some colour issues. Thanks!

![SS4 2](https://user-images.githubusercontent.com/33905827/226803707-8dc5e21f-22bc-44dc-b64f-0ac325a90dae.png)

![SS4 1](https://user-images.githubusercontent.com/33905827/226803759-4c35f929-e015-411f-92cc-c0f748ce1eec.png)

alya commented:
It looks like your PR changes how other error messages look, which was not the goal here.

<img width="279" alt="Screen Shot 2023-03-21 at 11 17 52 PM" src="https://user-images.githubusercontent.com/2090066/226818364-1cb204f5-34b5-4f02-98e2-d656ebf5ece7.png">
<img width="363" alt="Screen Shot 2023-03-21 at 11 17 33 PM" src="https://user-images.githubusercontent.com/2090066/226818367-30293de9-04cd-4040-8a5f-65a216ce84ac.png">

Nothell commented:
> It looks like your PR changes how other error messages look, which was not the goal here.
> 
> <img alt="Screen Shot 2023-03-21 at 11 17 52 PM" width="279" src="https://user-images.githubusercontent.com/2090066/226818364-1cb204f5-34b5-4f02-98e2-d656ebf5ece7.png">
> 
> <img alt="Screen Shot 2023-03-21 at 11 17 33 PM" width="363" src="https://user-images.githubusercontent.com/2090066/226818367-30293de9-04cd-4040-8a5f-65a216ce84ac.png">

Okay sure, I am attaching a screenshot of what I will be doing next please let me know if the error messages need to look like this.

**Current state:**
Light theme:
![SSCurrent](https://user-images.githubusercontent.com/33905827/227431959-c62c6498-60fc-402b-9850-e14fa7f8bf8d.png)

Dark theme:
![image](https://user-images.githubusercontent.com/33905827/227432077-c2c0d925-d7c7-42fa-baaa-a397ad44073a.png)


What I am aiming for:
I will match the colour on the personal setting's Full Name field with the error message shown below Profile Picture as shown in the screenshot below.

Ligh Theme:


Dark Theme:
![image](https://user-images.githubusercontent.com/33905827/227433536-ba9e8227-5a9a-4a2c-aa14-d54df97389b2.png)


![SSProposed](https://user-images.githubusercontent.com/33905827/227433895-c706a9ed-2172-43ef-9770-726fb59d1093.png)

Is this fine? Please let me know.

alya commented:
The error messages need to look just like they look now. You should verify that yourself, and proceed from there.

Nothell commented:
> I don't think we should move the profile picture just for the error message. But while keeping the placement below, why can't we make the font/colours the same as other error messages?

Thank you for your feedback. I understand that maintaining consistency is important, and I appreciate your guidance on this matter.

I want to make sure that I understand your expectations clearly. As of now, the error message UI in the personal settings section is inconsistent. Previously (as quoted above), you suggested that I make the font and colors consistent with other error messages, which I did. However, it seems that maintaining the current font and color is the desired outcome. To help resolve this issue, could you please confirm if my understanding is correct? I genuinely want to contribute and help in any way I can. Thank you for your guidance and patience.

alya commented:
Closing this PR as it makes unexpected changes beyond solving the issue at hand, so it's probably not the right direction for solving #17571.
